experimental/SetPartitions/src/PartitionProperties.jl

```diff
@@ -0,0 +1,194 @@
+function size(p::SetPartition)
+    return length(p.lower_points) + length(p.upper_points)
+end
+
+function is_composable(p::SetPartition, q::SetPartition)
+    return length(p.upper_points) == length(q.lower_points)
+end
+
+function upper_points(p::SetPartition)
```

Comment:
for my use-case, I would like to have accessors for the number of upper/lower points, e.g.
```julia
function num_upper_points(p::AbstractPartition)
	return length(upper_points(p))
end
```